pkg/csi/cinder/openstack/openstack_snapshots.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package openstack.snapshots provides an implementation of Snapshot related
+// cinder functions using Gohpercloud.
+package openstack
+
+import (
+	"github.com/golang/glog"
+	"github.com/gophercloud/gophercloud/openstack/blockstorage/v3/snapshots"
+)
+
+// CreateSnapshot issues a request to take a Snapshot of the specified Volume with the cooresponding ID and
```

Comment:
nit: check spelling for _ cooresponding_

Reply:
done